postgres-appliance/tests/docker-compose.yml

```diff
@@ -46,6 +46,7 @@ services:
                   postgresql:
                     parameters:
                       wal_keep_segments: 8
+                      jit: 'off'
```

Comment:
This is unreleated to the `bin_dir` changes, correct?

Reply:
it is the parameter that exists in PG11+ and causes 10->N upgrade failure if bin_dir is used incorrectly
However given we are getting rid of 10 (and likely 11) soon, I should probably add other parameters for regression testing in the future.

Reply:
Understood. Let's keep the parameter, especially given that PG10 is end-of-life by now and not be used much longer.